testutils/src/test/scala/za/co/absa/enceladus/testutils/rest/RestCallerTest.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.testutils.rest
+
+import org.scalatest.{BeforeAndAfterEach, FunSuite}
+import za.co.absa.enceladus.testutils.models.{RestMethod, TestCase, TestCaseResult}
+
+class RestCallerTest extends FunSuite with BeforeAndAfterEach {
+  test("testRun") {
+    val testCase: TestCase = TestCase("001",
+      "Test001",
+      "Post",
+      "application/json",
+      "https://httpbin.org/post",
```

Comment:
Does this mean that in order to pass, this test needs an open connection to the internet?